src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -187,4 +309,16 @@ private function isCommentLastLineToken(Tokens $tokens, $index)
 
         return $content !== ltrim($content, "\r\n");
     }
+
+    /**
+     * Checks if token is new line.
+     *
+     * @param Token $token
+     *
+     * @return bool
+     */
+    private function isNewLine(Token $token)
+    {
+        return $token->isWhitespace() && (false !== strpos($token->getContent(), "\n"));
```

Comment:
I know but I think it makes things a bit more clear. Should I remove them anyway?